packages/ag-charts-community/src/chart/test/examples.ts

```diff
@@ -2208,6 +2208,160 @@ export const STACKED_AREA_TIME_X_AXIS_MISSING_X_DATA_EXAMPLE: AgCartesianChartOp
     ],
 };
 
+const colorStops = [
+    {
+        stop: 0.2,
+        color: 'orange',
+    },
+    {
+        stop: 0.5,
+        color: 'red',
+    },
+    {
+        stop: 0.8,
+        color: 'blue',
+    },
+];
+
+export const AREA_SERIES_VERTICAL_GRADIENT_FILL: AgCartesianChartOptions = {
+    data: DATA_APPLE_REVENUE_BY_PRODUCT,
+    theme: {
+        overrides: {
+            area: {
+                series: {
+                    fill: {
+                        type: 'gradient',
+                        direction: 'vertical',
+                        colorStops,
+                    } as any, // not in the options for now
```

Comment:
Nit: Could you use a `// @ts-expect-error` so that once we do add this to the options, we'll be reminded to fix the type here?